### PR TITLE
docs: update pricing page to include service bindings section.

### PR DESCRIPTION
**Pull request by admah**
Added this to clarify pricing around service bindings.

### CODE REVIEW

File: content/workers/platform/pricing.md [hunk 1]
```diff
@@ -185,6 +185,12 @@ The [Durable Objects storage API](/workers/runtime-apis/durable-objects/#transac
 
 Requests that hit the [Durable Objects in-memory cache](/workers/learning/using-durable-objects/#accessing-persistent-storage-from-a-durable-object) or that use the [multi-key versions of get/put/delete methods](/workers/runtime-apis/durable-objects/#transactional-storage-api) are billed the same as if they were a normal, individual request for each key.
 
+## Service Bindings
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
## Service bindings
```

File: content/workers/platform/pricing.md [hunk 2]
```diff
@@ -185,6 +185,12 @@ The [Durable Objects storage API](/workers/runtime-apis/durable-objects/#transac
...
+
+Service Bindings cost the same as any normal Worker; each invocation is charged as if it’s a request from the Internet – with one major and important difference. You will be charged a single billable duration across all Workers triggered by a single incoming request.
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
Service bindings cost the same as any normal Worker. Each invocation is charged as if it is a request from the Internet with one important difference. You will be charged a single billable duration across all Workers triggered by a single incoming request.
```

Comment (deadlypants1973) [reply content/workers/platform/pricing.md]: Other languages don't use ; and - so to make readability and translation easier, we avoid these symbols in docs.

File: content/workers/platform/pricing.md [hunk 3]
```diff
@@ -185,6 +185,12 @@ The [Durable Objects storage API](/workers/runtime-apis/durable-objects/#transac
...
+
+For more information on how service bindings work, [read the docs](https://developers.cloudflare.com/workers/platform/bindings/about-service-bindings/).
```
Comment (deadlypants1973) [inline content/workers/platform/pricing.md]:
```suggestion
For more information on how service bindings work, refer to [About Service bindings](https://developers.cloudflare.com/workers/platform/bindings/about-service-bindings/).
```

**Review by deadlypants1973 — APPROVED** [review]
Approved with comment

File: content/workers/platform/pricing.md (hunk 2 above)
Comment (admah) [reply content/workers/platform/pricing.md]: I'll keep that in mind. Thanks!

**Pull request merged**